--- mtools-3.9.7/mtools.conf.big	Wed May 16 19:07:57 2001
+++ mtools-3.9.7/mtools.conf	Wed May 16 19:08:12 2001
@@ -2,8 +2,8 @@
 # your architecture and comment out the "SAMPLE FILE" line below
 
 # Linux floppy drives
-drive a: file="/dev/fd0" exclusive 1.44m mformat_only
-drive b: file="/dev/fd1" exclusive 1.44m mformat_only
+drive a: file="/dev/fd0" exclusive mformat_only
+drive b: file="/dev/fd1" exclusive mformat_only
 
 # First SCSI hard disk partition
 #drive c: file="/dev/sda1"
